lib/tests/sources/lparstat_source.cc

```diff
@@ -23,4 +22,12 @@ SCENARIO("Using the lparstat data source") {
         REQUIRE(lparstat_source.wpar_key() == 1);
         REQUIRE(lparstat_source.wpar_configured_id() == 1);
     }
+
+    WHEN("An lparstat executable is available but this is not an AIX machine") {
```

Comment:
This test looks like it's just making sure that the lparstat data is initialized correctly. I'd change the name to indicate that.